src/torchmetrics/image/fid.py

```diff
@@ -284,9 +295,16 @@ def __init__(
         feature: Union[int, Module] = 2048,
         reset_real_features: bool = True,
         normalize: bool = False,
+        input_img_size: Tuple[int, int, int, int] = (1, 3, 299, 299),
```

Comment:
we just need the user to provide the image dimension, the 1 batch dimension we can add internally
```suggestion
        input_img_size: Tuple[int, int, int] = (3, 299, 299),
```